src/lambdas/initiateAthenaQuery/handler.test.ts

```diff
@@ -174,4 +190,26 @@ describe('initiate athena query handler', () => {
     )
     expect(mockUpdateQueryByZendeskId).not.toHaveBeenCalled()
   })
+
+  it('returns an empty object as it is a manual query', async () => {
+    const testZendeskId = testManualAthenaQueryEvent.Records[0].body
+    const emailSNSTopicARN =
+      'arn:aws:sns:eu-west-2:123456789012:email-to-slack-topic'
+    when(getEnv)
```

Comment:
Our pattern for mocking environment variables is that, rather than mocking out the `getEnv` utility, we instead add a line to `src/utils/tests/setup/testEnvVars.ts`.